src/plugin/forum/Messenger/ForumNotification.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\ForumBundle\Messenger;
+
+use Claroline\AppBundle\Messenger\Message\AsyncInterface;
+use Claroline\ForumBundle\Entity\Message;
+
+class ForumNotification implements AsyncInterface
+{
+    /** @var Message */
+    private $message;
+
+    public function __construct(Message $message)
```

Comment:
```suggestion
    public function __construct(string $messageUuid)
```

Queued messages should not hold whole entities, only identifiers.
Then, the handler should be responsible for retrieving the entities it needs from the corresponding repositories.
(the job should be lightweight).
https://symfony.com/doc/current/messenger.html#doctrine-entities-in-messages
Reasons are that queued messages should be lightweight and we need to make sure handlers always deal with up-to-date entities.

Reply:
Done, thx :)